documentation/modules/kafkaconnectors/proc-deploying-kafkaconnector.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect.adoc
+
+[id='proc-deploying-kafkaconnector-{context}']
+= Deploying a `KafkaConnector` resource to Kafka Connect
+
+Deploy the example `KafkaConnector` to a Kafka Connect cluster. The example YAML file is configured to create a `FileStreamSourceConnector` to send each line of the Kafka license file as a message in Kafka.
+
+.Prerequisites
+
+* A Kafka Connect deployment in which xref:proc-enabling-kafkaconnectors-deployment-configuration-kafka-connect[`KafkaConnectors` are enabled]
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `examples/connector/source-connector.yaml` file:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: `KafkaConnector`
+metadata:
+  name: my-source-connector <1>
+  labels:   
+    strimzi.io/cluster: my-connect-cluster <2>
+spec:
+  class: org.apache.kafka.connect.file.FileStreamSourceConnector <3>
+  tasksMax: 2 <4>
+  config: <5>
+    file: "/opt/kafka/LICENSE"
+    topic: my-topic
+    # ...
+----
++
+<1> Enter a name for the `KafkaConnector` resource. You can choose any name.
+<2> Enter the name of the Kafka Connect cluster in which to create the connector.
+<3> The name or alias of the connector class.
+<4> The maximum number of tasks for the connector.
+<5> Configuration settings for the connector. Available configuration options depend on the connector class.
+
+. Create the `KafkaConnector` in your Kubernetes cluster:
++
+[source,shell,subs="+quotes"]
+----
+oc apply -f examples/connector/source-connector.yaml
+----
+
+.Additional resources
+
+* xref:proc-enabling-kafkaconnectors-deployment-configuration-kafka-connect[] 
+
+* xref:kafka-connect-{context}[] 
+
+* xref:type-KafkaConnect-reference[] 
+
+* xref:type-KafkaConnectS2I-reference[] 
```

Comment:
I wonder is links to these would be better in the prereqs? To get an understanding of what you can configure.